articles/libraries/standard/diagnostics.md

```diff
@@ -29,8 +29,8 @@ Message($"About to rotate by an angle of {angle}...");
 
 The <xref:microsoft.quantum.diagnostics.dumpmachine> and <xref:microsoft.quantum.diagnostics.dumpregister> callables instruct target machines to provide diagnostic information about all currently allocated qubits or about a specific register of qubits, respectively.
 Each target machine varies in what diagnostic information is provided in response to a dump instruction.
-The <xref:microsoft.quantum.simulation.simulators.quantumsimulator> target machine, for instance, provides the host program with the state vector that it uses internally to represent a register of qubits.
```

Comment:
This link and the link to Toffoli simulator still show up as broken at https://review.docs.microsoft.com/en-us/quantum/libraries/standard/diagnostics?branch=fixLinks

Reply:
I'm not surprised, we now have links to the .NET Core API browser up, but we also need to onboard more of the target machines into the API browser. Let me follow up with @anpaz-msft on how to best do that. Thanks for flagging this!